src/System.Runtime.CompilerServices.Unsafe/Directory.Build.props

```diff
@@ -1,10 +1,11 @@
 <Project>
 
-  <Import Project="..\Directory.Build.props" />
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props, $(MSBuildThisFileDirectory)..))" />
 
   <PropertyGroup>
     <VersionPrefix>6.0.0</VersionPrefix>
     <VersionPrefix Condition="'$(IsPackable)' == 'true'">6.1.0</VersionPrefix>
+    <AssemblyName>System.Runtime.CompilerServices.Unsafe</AssemblyName>

```

Comment:
If you set this here then the test project will also have that assembly name which is a bit weird. Consider inlining this property into the src project and the Versioning.csproj.

Reply:
I agree it should just be specified in Versioning.csproj, since that's the one that needs something different than the default assembly name.

Reply:
Resolved in the latest commit.